### PR TITLE
enable float type

### PR DESCRIPTION
**Pull request by hardikj**
This fixes https://phabricator.wikimedia.org/T75815

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -751,11 +751,13 @@ describe('DB backend', function() {
                 body: {
                     table: "typeTable",
                     proj: ['string','blob','set','int','varint', 'decimal',
-                            'double','boolean','timeuuid','uuid',
+                            'float', 'double','boolean','timeuuid','uuid',
                             'timestamp','json']
                 }
             })
             .then(function(response){
+                response.body.items[0].float = Math.round(response.body.items[0].float * 100) / 100;
```
Comment (gwicke) [inline test/index.js]: Minor: It would be nicer to define a 'roundDecimals' helper rather than repeating this inline.

**Comment by gwicke** [comment]
LGTM, but needs a rebase.

File: test/index.js (hunk 1 above)
Comment (hardikj) [inline test/index.js]: @gwicke A 'roundDecimals' helper should go in same file or in dbutils?

**Comment by hardikj** [comment]
Tests failed due to coverall failure

**Pull request merged**